scalafmt-dynamic/src/main/scala/org/scalafmt/dynamic/ScalafmtDynamic.scala

```diff
@@ -22,7 +24,7 @@ final case class ScalafmtDynamic(
     respectVersion: Boolean,
     respectExcludeFilters: Boolean,
     defaultVersion: String,
-    fmts: mutable.Map[Path, ScalafmtReflect]
+    fmts: mutable.Map[String, ScalafmtReflect]
```

Comment:
Now, the key of `fmts` is sha1 hash of the content of config of scalafmt.

Reply:
Can we avoid this change by creating a temporary file to host the string config contents?